derive_builder/src/options/field_mode.rs

```diff
@@ -125,23 +126,26 @@ impl From<OptionsBuilder<FieldMode>> for FieldOptions {
                     },
                     _ => syn::Ident::new(field_ident.clone()),
                 }});
+                
+        let setter_vis = b.setter_vis.unwrap_or(syn::Visibility::Public);
 
         FieldOptions {
             setter_enabled: b.setter_enabled.unwrap_or(true),
-            builder_pattern: b.builder_pattern.clone().unwrap_or_default(),
```

Comment:
@colin-kiegel I wasn't sure where all these `clone()` calls came from, but removing them didn't break any of the tests. There may not be any efficiency impact, but it's cleaner now :)